WashingBot.js

```diff
@@ -166,13 +168,13 @@ function setupMonitor(){
     }
 
     config.onFinished = function(washingDurationMinutes) {
-        txtStatus.SetText("Laundry finished!!! Duration: " + washingDurationMinutes + "\nWaiting for mulo...");
-        miniSlackBotInstance.sendMessage("Mulo a colgar la ropa!! :clock3: Duración del lavado: " + washingDurationMinutes);
+        txtStatus.SetText("Laundry finished!!! Duration: " + washingDurationMinutes + "\nWaiting for human...");
+        notifySlack("Human go hang the clothes!! :clock3: Washing duration: " + washingDurationMinutes);
         say("Laundry finished.");
     }
 
     config.onFinishedReminder = function(minsSinceFinish, reminderCount) {
-        miniSlackBotInstance.sendMessage(":information_source: " + reminderCount + " - Recordá colgar la ropa, hace " + minsSinceFinish + "mins. que terminó!");
+        notifySlack(":information_source: " + reminderCount + " - Remember to hang the clothes, washing has finished " + minsSinceFinish + "mins. ago!");
```

Comment:
I really don't know, it was like that before. Do you think it would be better with the space?
":information_source: 2 - Remember to hang the clothes, washing has finished 5mins. ago!"
":information_source: 2 - Remember to hang the clothes, washing has finished 5 mins. ago!"

It's pretty the same for me, although it seems better with the space IMO.